scripts/copy_client_assets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Tiny nit:  given that this script is only invoked from inside Makefiles, why not simply define it as a Makefile target (eg, put it in common.mk or something similar)?

Reply:
Done.